testGameSearch/GameService.swift

```diff
@@ -19,14 +19,16 @@ public class GameService {
     }
     */
     
-    func getGames(game: String, completion: @escaping (Game?) -> ()) {
+    func getGames(gameName: String, completion: @escaping (Game?) -> ()) {
         
-        let parameters = "fields name, summary, platforms.name; where platforms = (48,6,49); limit 10;"
+        let parameters = "search \"\(gameName)\"; fields name, summary, platforms.name; where platforms = (48,6,49); limit 10;"
```

Comment:
I am no expert with this API but it looks like the endpoint wants the search string in the HTTP post data rather than in the url.